cmake/configs/posix_rpi_common.cmake

```diff
@@ -8,6 +8,7 @@ include(posix/px4_impl_posix)
 add_definitions(
 	-D__PX4_POSIX_RPI
 	-D__DF_LINUX # For DriverFramework
+	-D__DF_RPI # For DriverFramework
```

Comment:
Perhaps we should make a note of it in that repo and come back to it. It seems to me that the DriverFramework cmake should be capable of determining this independently. Especially if there's any value in DF outside of PX4.